device-types/Cisco/C9800-80-K9.yaml

```diff
@@ -0,0 +1,49 @@
+---
+manufacturer: Cisco
+model: Catalyst 9800-80-K9
+slug: cisco-c9800-80-k9
+part_number: C9800-80-K9
+u_height: 2
+airflow: front-to-rear
+is_full_depth: false
+comments: '[Cisco Catalyst 9800-80 Wireless Controller Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9800-series-wireless-controllers/nb-06-cat9800-wirel-cont-data-sheet-ctp-en.html)'
+console-ports:
+  - name: console
+    type: rj-45
+  - name: usb0
+    type: usb-micro-b
+interfaces:
+  - name: Management Port
+    type: 1000base-t
+    mgmt_only: true
+  - name: Redundancy Ethernet Port
+    type: 1000base-t
+    mgmt_only: true
+  - name: Redundancy SFP Port
```

Comment:
In the configuration, what is the actual ports named? Is it actually called `Redundancy SFP Port`?

Reply:
Sorry, don't have access to a config at the moment, will try and find out the real name of this interface.

Reply:
Please let me know when you have found this information